examples/blog-refine-nextui/src/pages/products/edit.tsx

```diff
@@ -0,0 +1,171 @@
+  Textarea,
+} from "@nextui-org/react";
+
+import { ArrowLongLeftIcon } from "@heroicons/react/24/outline";
+
+import { IProduct, IProductCategory } from "../../interfaces";
+
+export const ProductEdit: React.FC<IResourceComponentsProps> = () => {
+  const goBack = useBack();
+
+  const {
+    refineCore: { onFinish, formLoading },
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm<IProduct, HttpError, IProduct>({
+    defaultValues: {
+      name: "",
+      description: "",
+    },
+  });
+
+  const { options: categoryOptions } = useSelect<IProductCategory>({
+    resource: "categories",
+  });
+
+  return (
+    <div>
+      <Card className="shadow-1 p-5">
+        <div className="flex items-center">
+          <Button
+            onClick={goBack}
+            className="m-1"
+            color="primary"
+            variant="light"
+            isIconOnly
+            aria-label="Go to products page"
+          >
+            <ArrowLongLeftIcon width={16} />
+          </Button>
+          <h1 className="text-lg font-bold">Edit Product</h1>
+        </div>
+        <form onSubmit={handleSubmit(onFinish)}>
+          <div className="p-fluid">
+            <Controller
+              name="name"
+              control={control}
+              rules={{ required: "Name is required." }}
+              render={({ field, fieldState }) => {
+                return (
+                  <Input
+                    label="Name"
+                    variant="bordered"
+                    labelPlacement="outside"
+                    placeholder="Enter product name"
+                    value={field.value ?? ""}
+                    errorMessage={errors[field.name]?.message ?? ""}
+                    type="text"
+                    onChange={(ev) => field.onChange(ev.target.value)}
+                  />
+                );
+              }}
+            />
+            <Controller
+              name="price"
+              control={control}
+              rules={{
+                required: "Price is required.",
+              }}
+              render={({ field, fieldState }) => {
+                return (
+                  <Input
+                    label="Price"
+                    variant="bordered"
+                    labelPlacement="outside"
+                    placeholder="Enter price"
+                    errorMessage={errors[field.name]?.message ?? ""}
+                    type="number"
+                    min="0"
+                    value={`${field.value ?? ""}`}
+                    onChange={(ev) => field.onChange(+ev.target.value)}
+                    className="mt-5"
+                  />
+                );
+              }}
+            />
+            <Controller
+              name="category.id"
+              control={control}
+              rules={{ required: "Category is required." }}
+              render={({ field, fieldState }) => {
+                console.log("Inside controller", field);
```

Comment:
Is this left on purpose?

```suggestion
                console.log("Inside controller", field);
```